src/MauiReactor/TemplateHost.cs

```diff
@@ -45,6 +45,11 @@ public VisualNode Root
     internal static void FireLayoutCycleExecuted(object? sender)
         => LayoutCycleExecuted?.Invoke(sender, EventArgs.Empty);
 
+    Microsoft.Maui.Controls.Page? IVisualNode.GetContainerPage()
+    {
+        return NativeElement as Microsoft.Maui.Controls.Page;

```

Comment:
not sure if this would ever be used, but decided to add it here so that all Host elements are consistent.